README.md

```diff
@@ -28,36 +31,15 @@ The name *Slint* is derived from our design goals:
     flexibility that only a native application can offer: Access full operating
     system APIs, utilize all CPU and GPU cores, connect to any peripheral.
 
-Visit [#MadeWithSlint](https://madewithslint.com) to view some of the projects
-using Slint. We invite you to use Slint and be part of its community.
-
-## Current Status
+Beyond the design goals, here’s what makes Slint stand out:
 
-Slint is in active development. The state of support for each platform is as
-follows:
+- **Design UI Independently**: Use a declarative language similar to separate your UI from business logic. Designers can work in parallel with developers.
```

Comment:
The use of a verb here while it's absent in the other bullet points feels a little inconsistent when reading. How about this?

```suggestion
- **Independent UI Design**: Use a declarative language similar to separate your UI from business logic. Designers can work in parallel with developers.
```